timesketch/lib/analyzers/feature_extraction_test.py

```diff
@@ -65,3 +69,125 @@ def test_config(self):
             self.assertIsInstance(key, str)
             self.assertIsInstance(value, dict)
             self._config_validation(value)
+
+    # Mock the Elasticsearch datastore.
+    @mock.patch(
+        'timesketch.lib.analyzers.interface.ElasticsearchDataStore',
+        MockDataStore)
+    def test_get_attribute_value(self):
+        """Test function _get_attribute_value()."""
+        analyzer = feature_extraction.FeatureExtractionSketchPlugin(
+            'test_index', 1)
+        #list, merge, multi
+        current_val = ['hello']
+        extracted_value = ['hello']
+        # pylint: disable=protected-access
+        new_val = analyzer._get_attribute_value(
+            current_val,
+            extracted_value,
+            True,
```

Comment:
to be more explicit and make it easier to read teh test can you rather do

```
new_val = analyzer._get_....(
    current_val=current_val,
    extracted_value=extracted_value,
    keep_multi=True,
    merge_values=True,
    type_list=True)

```

that way it will be easier to read the test